jackson-apt-processor/src/main/java/org/dominokit/jacksonapt/processor/DeserializerGenerator.java

```diff
@@ -25,30 +27,32 @@
      * generated too.
      *
      * @param beanType a {@link javax.lang.model.type.TypeMirror} object.
-     * @param packageName a {@link java.lang.String} object.
      * @return fully qualified deserialer name
      */
-    public String generate(String packageName, TypeMirror beanType) {
+    public String generate(TypeMirror beanType) {
+        String packageName = MoreElements.getPackage(MoreTypes.asTypeElement(beanType)).toString();
         String deserializerName = Type.deserializerName(packageName, beanType);
 
         if (!TypeRegistry.containsDeserializer(Type.stringifyTypeWithPackage(beanType))) {
             try {
-            	generateSubTypesDeserializers(beanType, packageName);
+                generateSubTypesDeserializers(beanType);
                 TypeRegistry.addInActiveGenDeserializer(beanType);
                 new AptDeserializerBuilder(packageName, beanType, filer).generate();
                 TypeRegistry.registerDeserializer(Type.stringifyTypeWithPackage(beanType), ClassName.bestGuess(deserializerName));
                 TypeRegistry.removeInActiveGenDeserializer(beanType);
+            } catch (FilerException e) {
+                //should be skipped
```

Comment:
Isnt this the same fix as in https://github.com/DominoKit/domino-jackson/pull/39

Reply:
crap, yeap, you right, i need to remove it

Reply:
This is not resolved yet, right?!